src/store.js

```diff
@@ -78,6 +80,17 @@ export class Store {
     }
   }
 
+  applyMixins (options) {
+    if (Array.isArray(options.mixins)) {
+      const merge = Vue.config.optionMergeStrategies.computed
+      options.mixins.forEach(mixin => {
+        for (const key in mixin) {
+          options[key] = merge(mixin[key], options[key])
```

Comment:
Since the `options` object is mutated, there would be a problem if the module is reused in multiple times. i.e. it reapplies the mixins for the already applied options object.